tests/llm/utils/tool_suggestions_config.py

```
@@ -0,0 +1,275 @@
+import os
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, Tuple
+
+from holmes.core.tools_utils.frontend_tools import build_frontend_noop_tool
+
+SUGGEST_RUNBOOKS_TOOL_NAME = "suggest_runbooks"
+
+
+# Description shown to the LLM as the tool's description.
+SUGGEST_RUNBOOKS_TOOL_DESCRIPTION = (
+    f"Call this tool at the end of your response if the conversation contains "
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)